### ISSUE TITLE
Minimum tool level

### ISSUE THREAD

**Issue by JWTHDYTWA** [codetaylor/artisan-worktables#44]
What about the requirement for a sufficiently durable tool in recipes? I think it's unacceptable to have workaround like using up one wooden hammer for recipe that requires over900 durability.

**Comment by codetaylor** [comment]
I've included a config option that will be available in the next feature update:

```
# If set to true, crafting tools must have sufficient durability remaining to perform the craft.
# If set to false, this restriction is ignored.
B:RESTRICT_CRAFT_MINIUMUM_DURABILITY=true
```

By default this will be set to `true`.

**Comment by codetaylor** [comment]
Available in 1.12.2-1.11.20
